lib/plugin/screenshotOnFail.js

```diff
@@ -75,16 +75,17 @@ module.exports = function (config) {
   event.dispatcher.on(event.test.failed, (test) => {
     recorder.add('screenshot of failed test', async () => {
       let fileName = clearString(test.title);
+      const imageExtension =  options.imageExtension ? options.imageExtension : 'png';
```

Comment:
Shall we handle the case which user provides the invalid extension?

I guess it's better to create a list of supporting image extensions and basically we loop through that list.

Reply:
Get your point, but I wonder if the list of supported extensions might be platform dependant, I couldn't find one at all to tell the truth, I just entered some by chance on my docker images and OSX locally.
As resemble.js allows to even save an image without extension at all (forgot that once on a filename) there doesn't seem to be much checks in the underlying tools.

Reply:
So what if I provide something like ffxkf as image extension? 😅